src/lib/generateSchemaJson.ts

```
@@ -1,31 +1,50 @@
+  }
+
+  return formatJson(properties)
 }
 
-export function propertiesTypesJsonRecursive(schema: any, useExample = false) {
+export function propertiesTypesJsonRecursive(schema: any, useExample = false, visited = new Set(), level = 0) {
+  if (visited.has(schema)) {
+    if (level > 10) {
+      return '[Circular Reference]'
+    }
+    visited = new Set()
```

Biome

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)